ansys-grantami-serverapi-openapi/src/ansys/grantami/serverapi_openapi/__init__.py

```diff
@@ -41,10 +47,13 @@
 from .api.schema___tables_api import SchemaTablesApi
 from .api.schema___units_api import SchemaUnitsApi
 from .api.search_api import SearchApi
+from .api.selection_searches_api import SelectionSearchesApi
 from .api.status_api import StatusApi
 
 # import models into sdk package
 from .models.folderguid_files_body import FolderguidFilesBody
+from .models.folderguid_files_body1 import FolderguidFilesBody1
+from .models.folderguid_files_body2 import FolderguidFilesBody2
```

Comment:
Being investigated

Reply:
Is this a problem? I assume the code references the models correctly?

Reply:
No. But I think the endpoints are not well documented, I need to have a better look